library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -3,10 +3,10 @@ package scala.runtime.stdLibPatches
 object Predef:
   import compiletime.summonFrom
 
-  inline def assert(inline assertion: Boolean, inline message: => Any): Unit =
+  transparent inline def assert(inline assertion: Boolean, inline message: => Any): Unit =
     if !assertion then scala.runtime.Scala3RunTime.assertFailed(message)
```

Comment:
Seems it can be inlined later? The same for below.

Reply:
This one has a special interaction with null safety that requires it to be inlined in typer. It would be much better to inline after typer.